test/unit/puppetlabs/services/request_handler/request_handler_core_test.clj

```diff
@@ -121,7 +121,8 @@
   (testing "request with binary content type does not consume body"
     (let [body-string "some random text"]
       (doseq [content-type [nil "" "application/octet-stream"
-                            "APPLICATION/octet-Stream"]]
+                            "APPLICATION/octet-Stream"
```

Comment:
If this has a more concrete integration test it might be worth extending test coverage to actually making a HTTP request, but that might be more work than it's worth.